sdk/canton/community/base/src/main/scala/com/digitalasset/canton/data/ActionDescription.scala

```diff
@@ -381,6 +384,7 @@ object ActionDescription extends HasProtocolVersionedCompanion[ActionDescription
     override def pretty: Pretty[ExerciseActionDescription] = prettyOfClass(
       param("input contract id", _.inputContractId),
       param("template id", _.templateId),
+      param("interface id", _.interfaceId),
```

Comment:
paramIfDefined ?

Reply:
Ah, I haven't done this on 2x, as there were other optional fields that also just showed "None" explicitly when missing
I can update it here, but it'll be inconsistent

Reply:
If relevant, I'd change for paramIfDefined: inconsistency is not an issue here